src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -940,8 +944,10 @@ FixedwingPositionControl::control_auto(const float control_interval, const Vecto
 void
 FixedwingPositionControl::control_auto_fixed_bank_alt_hold(const float control_interval)
 {
-	// only control altitude and airspeed ("fixed-bank loiter")
+	// check if generic low-height flight conditions are satisfied
```

Comment:
```suggestion
```
I think `checkLowHeightConditions` is explicit enough, if not you should rather improve the function name than adding a comment